src/commands/lambda/__tests__/instrument.test.ts

```diff
@@ -1318,7 +1318,7 @@ UpdateFunctionConfiguration -> arn:aws:lambda:sa-east-1:123456789012:function:la
 }
 TagResource -> arn:aws:lambda:sa-east-1:123456789012:function:lambda-hello-world
 {
-  \\"dd_sls_ci\\": \\"v1.2.0\\"
+  \\"dd_sls_ci\\": \\"v1.3.0\\"
```

Comment:
A bit tedious to update every time, we should look for a way to mock the version there in the future

Reply:
agreed. Would it be as simple as reading `package.json` from the root?

Reply:
Yes, it seems it was already done in some tests. Created a small PR to update it: https://github.com/DataDog/datadog-ci/pull/520

Reply:
nice, thanks